apps/voting/contracts/Voting.sol

```diff
@@ -459,12 +462,14 @@ contract Voting is IForwarder, AragonApp {
     }
 
     /**
-    * @dev Internal function to check if a voter can at least object to a vote outcome. It assumes the queried vote exists.
-    * @return True if the given voter can object to a certain vote outcome, false otherwise
+    * @dev Internal function to get the current phase for the vote. It assumes the queried vote exists.
+    * @return VotingPhase.Main if one can vote yes or no and VotingPhase.Objection if one can vote only no
     */
-    function _canObject(uint256 _voteId, address _voter) internal view returns (bool) {
-        Vote storage vote_ = votes[_voteId];
-        return _isVoteOpenForObjection(vote_) && token.balanceOfAt(_voter, vote_.snapshotBlock) > 0;
+    function _getVotingPhase(Vote storage vote_) internal view returns (VotingPhase) {
+        if (getTimestamp64() < vote_.startDate.add(voteTime).sub(objectionTime)) {
+            return VotingPhase.Main;
+        }
+        return VotingPhase.Objection;
```

Comment:
Should prob return smth like `VotePhase.Closed` if vote not active anymore

Reply:
Good point

Reply:
Maybe `VotePhase.ENDED`?